arctic/tickstore/tickstore.py

```diff
@@ -343,7 +343,7 @@ def read(self, symbol, date_range=None, columns=None, include_images=False, allo
             rtn = rtn.sort_index(kind='mergesort')
         if date_range:
             # FIXME: support DateRange.interval...
-            rtn = rtn.ix[date_range.start:date_range.end]
+            rtn = rtn[date_range.start:date_range.end]
```

Comment:
As pandas slicing is quite overloaded was wondering if it should be more specific to the particular index.

Not opinionated though, as long as it works and perf is good